app/controllers/coronavirus_form/live_in_england_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::LiveInEnglandController < ApplicationController
+  include ActionView::Helpers::SanitizeHelper
+  include FieldValidationHelper
+
+  def show
+    render "coronavirus_form/#{PAGE}"
+  end
+
+  def submit
+    live_in_england = sanitize(params[:live_in_england]).presence
+    session[:live_in_england] = live_in_england
+
+    invalid_fields = validate_radio_field(
+      PAGE,
+      radio: live_in_england,
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      log_validation_error(invalid_fields)
+      render "coronavirus_form/#{PAGE}", status: :unprocessable_entity
+    elsif session["check_answers_seen"]
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    else
```

Comment:
If no, we chuck them out, according to the spreadsheet. There's none of that in here.

Reply:
Yeah, that page is still being built.